sketcherMinimizer.h

```diff
@@ -463,10 +445,81 @@ class EXPORT_COORDGEN sketcherMinimizer
     void setTreatNonterminalBondsToMetalAsZOBs(bool b) {m_treatNonterminalBondsToMetalAsZOBs = b;}
 
+    /* add bond to m_extrabonds */
+    void addExtraBond(sketcherMinimizerBond* bond);
+
+    /* getters */
+    std::vector<sketcherMinimizerAtom*> getAtoms() { return _atoms; }
+    std::vector<sketcherMinimizerMolecule*> getMolecules() { return _molecules; }
+    std::vector<sketcherMinimizerAtom*> getReferenceAtoms() { return _referenceAtoms; }
+    std::vector<sketcherMinimizerBond*> getReferenceBonds() { return _referenceBonds; }
+    std::vector<sketcherMinimizerStretchInteraction*> getStretchInteractions();
+    std::vector<sketcherMinimizerInteraction*> getInteractions();
+
+    /* setters */
+    void setFragments(std::vector<sketcherMinimizerFragment*> fragments) { _fragments = fragments; }
+    void setEvenAngles(bool b);
+    void setSkipMinimization(bool b);
+    void setForceOpenMacrocycles(bool b);
+
+    /// Interactions with CoordgenMinimizer and CoordgenFragmentBuilder
+    /* find a list of carbons from the backbone C=O of a peptide */
+    std::set<sketcherMinimizerAtom*>
+    getChetoCs(const std::vector<sketcherMinimizerAtom*>& allAtoms);
+
+    /* find a list of nitrogens from the backbon NH of a peptide */
+    std::set<sketcherMinimizerAtom*>
+    getAminoNs(const std::vector<sketcherMinimizerAtom*>& allAtoms);
+
+    /* find a list of alpha carbons of a peptide */
+    std::set<sketcherMinimizerAtom*>
+    getAlphaCs(const std::vector<sketcherMinimizerAtom*>& allAtoms,
+               const std::set<sketcherMinimizerAtom*>& chetoCs,
+               const std::set<sketcherMinimizerAtom*>& aminoNs);
+
+    /* clear all the interactions loaded in the minimizer and free memory */
+    void clearInteractions();
+
+    /* setup all constraints of given molecule */
+    void addInteractionsOfMolecule(sketcherMinimizerMolecule* molecule,
+                                   bool intrafragmentClashes = false);
+
+    /* run a force-field based minimization on the given molecule */
+    void minimizeMolecule(sketcherMinimizerMolecule* molecule);
+
+    /* run a force-field based minimization */
+    void run();
+
+    void addExtraInteraction(sketcherMinimizerMolecule* molecule,
+                             sketcherMinimizerInteraction* interaction);
+
+    void buildFromFragments(bool b);
+    bool avoidClashesOfMolecule(
+        sketcherMinimizerMolecule* molecule,
+        const std::vector<sketcherMinimizerInteraction*>& extraInteractions =
+            std::vector<sketcherMinimizerInteraction*>());
+
 private:
     /*all non-terminal bonds to a metal atom are treated as if they were zero order bonds (this usually results
      in a longer bond*/
     bool m_treatNonterminalBondsToMetalAsZOBs = true;
+
+    CoordgenFragmentBuilder m_fragmentBuilder;
+    CoordgenMinimizer m_minimizer;
+
+    std::vector<sketcherMinimizerAtom*> _atoms;
```

Comment:
can we take the chance to be consistent and either use _member or m_member? my vote is on m_member

Reply:
Let's standardize on `m_member` where we can.

Reply:
that makes sense, I updated that